### DISCUSSION TITLE
v1.0.0-beta.3

### DISCUSSION THREAD

**Discussion by EvanMulawski** [EvanMulawski/BeamMP.EnvSync#15] [Announcements]
- Fixes "You do not have permissions to do this" error (#14)
  - Thanks to Spleens1 for the report

<hr /><em>This discussion was created from the release <a href='https://github.com/EvanMulawski/BeamMP-EnvSync/releases/tag/v1.0.0-beta.3'>v1.0.0-beta.3</a>.</em>

**Comment by Sgt-PieFace** [comment]
I am having trouble getting permission to use /env set time through in game chat.

These are the lines in the envsync.config.lue I have tried: 
-- options.admins.pieguy_99.json = "welcome admin"
-- options.admins.beammp_123456 = "pieguy_99"
-- options.admins.beammp_123456 = "https://forum.beammp.com/u/<pieguy_99>.json"
-- options.admins = {pieguy_99}

My server console says it is loading 1 admin but I still have no in gamer permissions with the following enabled:
-- options.admins.beammp_123456 = "pieguy_99"
-- options.admins.beammp_123456 = "https://forum.beammp.com/u/<pieguy_99>.json"

when trying to load https://forum.beammp.com/u/<pieguy_99>.json in chrome all I get on the page is {"errors":["The requested URL or resource could not be found."],"error_type":"not_found"}

I'm sure I'm missing something but I'm lost and any help is appreciated!

**Reply by EvanMulawski** [reply]
No carets: https://forum.beammp.com/u/pieguy_99.json

The `--` at the start of the line means the line does not take effect (a comment line), so remove those.

So you want:

```lua
options.admins.beammp_734609 = "pieguy_99 (or any other text you want here)"
```
